### ISSUE TITLE
[Accessibility] A11y_Manage.IIS.Net_Access Token_Screen Reader: Narrator/NVDA is not reading the labels of the input elements present on the page.

### ISSUE THREAD

**Issue by accessibility18** [Microsoft/IIS.WebManager#184]
**Environment Details:**
URL: https://manage.iis.net/
Browser: Microsoft Edge 42.17134.1.0, Google Chrome Version 68.0.3440.106
Screen Reader: Narrator, NVDA

**MAS ID Violated:** MAS1.3.1

**Repro Steps:**
1. Open "Microsoft IIS" application using the URL: https://manage.iis.net/
2. Download and install "Microsoft IIS Administration" application.
3. Tab till "Skip This" button and Press Enter.
4. Tab till "Get Access Token" link and Press Enter
5. Tab till "Create Access Key" button and Press Enter.
6. Tab till "Create " button and Press Enter.
7. Verify that screen reader is reading correct control type of all the UI elements.

**Actual Result:**
Narrator/NVDA is reading "Got It" button as "link". Visually it appears as button 

**Expected Result:**
"Got It" is a button and it does not actually serve the purpose of the link. So, it's role should be provided as button. Screen Reader should also read it as button.

**User Impact:**
Screen Reader users will find it difficult to understand the result of the user action after selecting the "Got It" control.

**MAS Reference:**
https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

**Attachment:**
[Bug 184_Screen reading is reading button as link.pptx](https://github.com/Microsoft/IIS.WebManager/files/2335517/Bug.184_Screen.reading.is.reading.button.as.link.pptx)

**Comment by jhkimnew** [comment]
duplicate bug to https://github.com/Microsoft/IIS.WebManager/issues/236

**Comment by jhkimnew** [comment]
This UI is not used anymore.

**Comment by mstechie** [comment]
GitHubTags:#A11y_Manage.IIS.Net;
